src/routes/[lang=lang]/+page.svelte

```diff
@@ -7,18 +7,19 @@
 	let spectators = 0
 
 	onMount(() => {
-		const interval = setInterval(updateSpectatorCount, 2_000)
+		const interval = setInterval(async () => {
+			spectators = await updateSpectatorCount(spectators)
+		}, 2_000)
 
 		return () => clearInterval(interval)
 	})
 
-	const updateSpectatorCount = () => {
-		// no real data, just a simulation ;)
-		spectators = spectators * 2 + 1
-
-		if (spectators > 100_000) {
-			spectators = 0
-		}
+	const updateSpectatorCount = async (spectators: number) => {
```

Comment:
You don't need to pass `spectators` since the function can already use it. Not the best functional approach but for such a small example, I would prefer to not pass a lot of data around.